tensorflow/lite/micro/kernels/leaky_relu.cc

```diff
@@ -28,55 +28,59 @@ namespace micro {
 namespace activations {
 namespace {
 
+// Input/output tensor index.
+constexpr int kInputTensor = 0;
+constexpr int kOutputTensor = 0;
+
 // OLD-TODO(b/142762739): We should figure out a multi-threading plan for most
 // of the activation ops below.
 
-struct OpData {};
-
-struct LeakyReluOpData : public OpData {
-  int32_t output_multiplier_alpha = 0;
-  int32_t output_shift_alpha = 0;
-  int32_t output_multiplier_identity = 0;
-  int32_t output_shift_identity = 0;
+struct LeakyReluOpData {
+  // quantization parameters
+  int32_t output_multiplier_alpha;
+  int output_shift_alpha;
+  int32_t output_multiplier_identity;
+  int output_shift_identity;
+  int32_t input_zero_point;
+  int32_t output_zero_point;
 };
 
 template <typename T>
-void QuantizeLeakyRelu(const TfLiteTensor* input, TfLiteTensor* output,
+void QuantizeLeakyRelu(const TfLiteEvalTensor* input, TfLiteEvalTensor* output,
                        const LeakyReluOpData* data) {
```

Comment:
The TfLite code doesn't always follow the Google style guide and we are incrementally moving towards better conformance within TFLM.


Reorder the params to be inputs then outputs. And consider changing the params to be const and non-const references (this may be awkward given the existing APIs, but do it if possible).

Same suggestion for the other (non-kernel API) functions in this kernel and the test as well.

https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs

Reply:
Fixed